Nef_3/include/CGAL/Nef_3/ID_support_handler.h

```diff
@@ -78,12 +82,13 @@ class ID_support_handler<SNC_indexed_items, Decorator> {
     hash[get_hash(i)] = parent;
   }
 
+  void initialize_hash(int i) {
```

Comment:
Did you intentionally move this overload before the function template?  If not, let's move it back in order to have a smaller diff.

Reply:
I guess the order of declaration doesn't matter in a C++ class, but I thought from a reader's perspective it's better to have the dependant method first, and I made the template method depend on this one.

Reply:
Fine with me, as t is intentional.